server/index.js

```diff
@@ -5,7 +5,7 @@ const axios = require('axios')
 const BN = require('bn.js')
 const parseString = require('xml2js').parseString
 const app = express()
-const port = process.env.PORT || 8000
+const port = process.env.PORT || 7000
```

Comment:
Change that please to 8000 instead of 7000, as it will break the istio network config in the kubernetes cluster